lib/pushmi_pullyu/cli.rb

```diff
@@ -26,12 +27,19 @@ def parse(argv = ARGV)
   end
 
   def run
-    if options[:daemonize]
-      start_server_as_daemon
-    else
-      # If we're running in the foreground sync the output.
-      $stdout.sync = $stderr.sync = true
-      start_server
+    configure_rollbar
+    begin
+      if options[:daemonize]
+        start_server_as_daemon
+      else
+        # If we're running in the foreground sync the output.
+        $stdout.sync = $stderr.sync = true
+        start_server
+      end
+    # rubocop:disable Lint/RescueException
```

Comment:
The problem is, that will mean certain kinds of non-standarderror exceptions won't get reported to Rollbar. It's good advice in general, just not when you actually do want a failsafe to catch _everything_, so you can report it.

![everyone](http://2.bp.blogspot.com/-In5VWi5P0cA/Uhz5L_UYI-I/AAAAAAAAGQg/xTHljS50f9o/s1600/The+Professional+Gary+Oldman+everyone.gif)